packages/samples/headless-ssr-commerce/app/_components/recommendation-list.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  Recommendations as RecommendationsController,
+  RecommendationsState,
+} from '@coveo/headless/ssr-commerce';
+import {useEffect, useState, FunctionComponent} from 'react';
+
+interface RecommendationsProps {
+  staticState: RecommendationsState;
+  controller?: RecommendationsController;
+}
+
+export const Recommendations: FunctionComponent<RecommendationsProps> = ({
+  staticState,
+  controller,
+}) => {
+  const [state, setState] = useState(staticState);
+
+  useEffect(
+    () => controller?.subscribe(() => setState({...controller.state})),
+    [controller]
+  );
+
+  return (
+    <>
+      <h3>{state.headline}</h3>
+      <ul className="product-list">
+        {state.products.map((product) => (
+          <li key={product.ec_product_id}>
+            <h3>{product.ec_name}</h3>
+          </li>
+        ))}
+      </ul>
+      <br />
+    </>
+  );
```

Comment:
Do you mind making this clickable like the product list so it gets redirected to the product-page?